scaleway/resource_rdb_instance.go

```diff
@@ -321,7 +321,7 @@ func resourceScalewayRdbInstanceCreate(ctx context.Context, d *schema.ResourceDa
 		}
 
 		_, err = waitInstance(ctx, rdbAPI, region, res.ID)
-		if err != nil && !is404Error(err) {
+		if err != nil {
```

Comment:
Why are those changes needed? Should this kind of change be present on all other resources that check for 404 errors?

Reply:
After a review of code, I remove is404Error where I consider where I think is necessary that the instance must exist. Which protect other request with an InstanceID not null and in a stable mode.